Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -268,6 +268,12 @@ var ListView = React.createClass({
     }
   },
 
+  getScrollableNode: function() {
+    if (this._scrollComponent && this._scrollComponent.getScrollableNode) {
+      return this._scrollComponent.getScrollableNode();
+    }
```

Comment:
Let's add an else case here and return `findNodeHandle(this._scrollComponent)` so we always return a node even if the scroll component doesn't implement getScrollableNode.